### PR TITLE
Update REI group of Chipped mod

### PR DESCRIPTION
**Pull request by toby7002**
Move workbench and tables blocks out of the group which can improve a little bit of UX (players don't need to `Alt` + Click)

### CODE REVIEW

File: kubejs/client_scripts/script.js
Comment (Erdragh) [inline kubejs/client_scripts/script.js:1]: It seems you forgot to include the `chipped:mechanist_workbench`

**Review by Erdragh — CHANGES_REQUESTED** [review]
Looks and works fine in game, but you should add the Tinkerer's Workbench to the exclusions as well

**Comment by toby7002** [comment]
> Looks and works fine in game, but you should add the Tinkerer's Workbench to the exclusions as well

Solved!

**Review by Erdragh — APPROVED** [review]
From my POV this is ready to be merged. Let's see what @Laskyyy thinks about it

**Pull request closed without merging**